src/commands/math/basicSymbols.js

```diff
@@ -485,3 +486,173 @@ LatexCmds['÷'] = LatexCmds.div = LatexCmds.divide = LatexCmds.divides =
+LatexCmds.gtreqless = LatexCmds['⋛'] = bind(VanillaSymbol, '\\gtreqless ', '&#8923;');
+LatexCmds.curlyeqprec = LatexCmds['⋞'] = bind(VanillaSymbol, '\\curlyeqprec ', '&#8926;');
+LatexCmds.curlyeqsucc = LatexCmds['⋟'] = bind(VanillaSymbol, '\\curlyeqsucc ', '&#8927;');
+LatexCmds.nsim = LatexCmds['≁'] = bind(VanillaSymbol, '\\nsim ', '&#8769;');
+LatexCmds.lnsim = LatexCmds['⋦'] = bind(VanillaSymbol, '\\lnsim ', '&#8934;');
+LatexCmds.gnsim = LatexCmds['⋧'] = bind(VanillaSymbol, '\\gnsim ', '&#8935;');
+LatexCmds.precnsim = LatexCmds['⋨'] = bind(VanillaSymbol, '\\precnsim ', '&#8936;');
+LatexCmds.succnsim = LatexCmds['⋩'] = bind(VanillaSymbol, '\\succnsim ', '&#8937;');
+LatexCmds.ntriangleleft = LatexCmds['⋪'] = bind(VanillaSymbol, '\\ntriangleleft ', '&#8938;');
+LatexCmds.ntriangleright = LatexCmds['⋫'] = bind(VanillaSymbol, '\\ntriangleright ', '&#8939;');
+LatexCmds.ntrianglelefteq = LatexCmds['⋬'] = bind(VanillaSymbol, '\\ntrianglelefteq ', '&#8940;');
+LatexCmds.ntrianglerighteq = LatexCmds['⋭'] = bind(VanillaSymbol, '\\ntrianglerighteq ', '&#8941;');
+LatexCmds.blacksquare = LatexCmds['∎'] = bind(VanillaSymbol, '\\blacksquare ', '&#8718;');
+LatexCmds.colon = LatexCmds['∶'] = bind(VanillaSymbol, '\\colon ', '&#8758;');
+LatexCmds.llcorner = LatexCmds['∟'] = bind(VanillaSymbol, '\\llcorner ', '&#8735;');
+
+// New BinaryOperators
+LatexCmds.dotplus = LatexCmds['∔'] = bind(BinaryOperator, '\\dotplus ', '&#8724;');
+LatexCmds.nmid = LatexCmds['∤'] = bind(BinaryOperator,'\nmid ','&#8740;');
+LatexCmds.intercal = LatexCmds['⊺'] = bind(BinaryOperator, '\\intercal ', '&#8890;');
+LatexCmds.veebar = LatexCmds['⊻'] = bind(BinaryOperator, '\\veebar ', '&#8891;');
+LatexCmds.barwedge = LatexCmds['⊼'] = bind(BinaryOperator, '\\barwedge ', '&#8892;');
+LatexCmds.ltimes = LatexCmds['⋉'] = bind(BinaryOperator, '\\ltimes ', '&#8905;');
+LatexCmds.rtimes = LatexCmds['⋊'] = bind(BinaryOperator, '\\rtimes ', '&#8906;');
+LatexCmds.leftthreetimes = LatexCmds['⋋'] = bind(BinaryOperator, '\\leftthreetimes ', '&#8907;');
+LatexCmds.rightthreetimes = LatexCmds['⋌'] = bind(BinaryOperator, '\\rightthreetimes ', '&#8908;');
+LatexCmds.curlyvee = LatexCmds['⋎'] = bind(BinaryOperator, '\\curlyvee ', '&#8910;');
+LatexCmds.curlywedge = LatexCmds['⋏'] = bind(BinaryOperator, '\\curlywedge ', '&#8911;');
+LatexCmds.circledcirc = LatexCmds['⊚'] = bind(BinaryOperator, '\\circledcirc ', '&#8858;');
+LatexCmds.circledast = LatexCmds['⊛'] = bind(BinaryOperator, '\\circledast ', '&#8859;');
+LatexCmds.circleddash = LatexCmds['⊝'] = bind(BinaryOperator, '\\circleddash ', '&#8861;');
+LatexCmds.boxplus = LatexCmds['⊞'] = bind(BinaryOperator, '\\boxplus ', '&#8862;');
+LatexCmds.boxminus = LatexCmds['⊟'] = bind(BinaryOperator, '\\boxminus ', '&#8863;');
+LatexCmds.boxtimes = LatexCmds['⊠'] = bind(BinaryOperator, '\\boxtimes ', '&#8864;');
+LatexCmds.boxdot = LatexCmds['⊡'] = bind(BinaryOperator, '\\boxdot ', '&#8865;');
+
+// New mappings to existing symbols
```

Comment:
Would it be better to put these aliases in the same files that the original commands are declared, or is keeping them all in this file preferred?

Reply:
Yeah I thought about that but I'd prefer to keep them here so they're not spread all over the code, as it's kind of a hacky change.

Reply:
That's what I was figuring. :+1: